cmd/juju/controller/addmodel.go

```diff
@@ -396,6 +398,16 @@ there is no default cloud defined, please specify one using:
 	return cloudTag, cloud, nil
 }
 
+var ambiguousDetectedCredentialError = errors.New(`
+more than one credential detected
```

Comment:
We should tell the user more info about the detected credentials, eg for aws, the name of the credential from the ini file.
Otherwise, they have no clue what argument to use with --credential. Or at the least, tell them how to find about what credentials have been detected